magefile.go

```diff
@@ -32,7 +32,8 @@ import (
 
 	devtools "github.com/elastic/beats/v7/dev-tools/mage"
 	"github.com/elastic/beats/v7/dev-tools/mage/gotool"
-	testtools "github.com/elastic/beats/v7/dev-tools/mage/target/unittest"
+        // mage:import
+        "github.com/elastic/beats/v7/dev-tools/mage/target/unittest"
```

Comment:
Nit: This looks like it has an extra level of indentation. Is that intentional?

Reply:
This is probably causing the linting failures.